api/v1alpha/framework/xcuitest.schema.json

```diff
@@ -174,6 +174,10 @@
           },
           "smartRetry": {
             "$ref": "../subschema/common.schema.json#/definitions/smartRetry"
+          },
+          "shardConfig": {
+            "description": "Path to the sharding configuration file which contains XCUITest tests",
```

Comment:
This makes it sound so complex (rename 🙏 ) 😅 
It's not a sharding configuration file though. It's merely a file with a list of test classes (separated by a new line). Might be a good info to add here.

Reply:
Additionally, any strong reason not to make use of the existing flag/field we have for other frameworks, the one that's called `shard`? It can certainly have its own values that differ from other frameworks (e.g. `spec` isn't a thing in xcuitest), but `shard: concurrency` would still work and the list of classes/methods would come from the given file.

Reply:
`shard: concurrency` also looks good to me 😂 Just wanted to merge them into one. I'll update them accordingly.